week1/09.py

```diff
@@ -0,0 +1,4 @@
+a = [10,20,30,20,10,50,60,40,80,50,40]
+reduced_a=[]
+[reduced_a.append(x) for x in a if x not in reduced_a]
+print(reduced_a)
```

Comment:
Here in line 6 you are using a list concatenation, which produces a new list. But this new list is never written into a variable. This suggest that you are using list concatenation wrong.

Two ways of rewriting you code:

With an actual for loop
```python
res = []
for x in a:
    if x not in res:
        res.append(x)
```
Or with an actual list concatenation:
```python
res = [x for i,x in enumerate(a) if i%2 == 0 ]
```